src/EventListener/LogRevisionsListener.php

```diff
@@ -389,6 +426,49 @@ private function getInsertRevisionSQL(ClassMetadata $class): string
         return $this->insertRevisionSQL[$class->name];
     }
 
+    /**
+     * @param ClassMetadata $class
```

Comment:
```suggestion
     * @param ClassMetadata<object> $class
```
avoid the need of psalm-param